Proxy/proxy.py

```diff
@@ -448,6 +565,7 @@ def sqlInsert(data):
 
 def main():
     log = proxyConfig.getboolean('PROXY', 'LOG')
+    testmode = proxyConfig.getboolean('PROXY', 'TESTMODE')
```

Comment:
I saw it done in other places but followed suit on this occasion.  I'm gonna touch the files anyhow so I'll do it for these two config reads.